src/Reducer/Reducing.js

```diff
@@ -0,0 +1,24 @@
+import { combineReducers } from 'redux'
+
+const taskReducer = (state = [], action) => {
+    switch (action.type) {
+        case 'ADD_TODO':
+            console.log(action.text)
+            state = state.concat(action.text)
+
+            break;
+        case 'DEL_TODO':
+            var x = state.indexOf(action.del)
+            state = state.slice()
```

Comment:
Why did you first use slice and then splice?
And its bad practice to modify the functional argument, better create a local object and return that.
Also in this case you have not returned anything, it works because splice changes the reference.